### PR TITLE
added work model for artwork pieces

### PR DESCRIPTION
**Pull request by allisondurham**
take a look at this, added a model for art pieces, called "Work"  , i think its ready for merging

### CODE REVIEW

File: places/models.py [hunk 1]
```diff
@@ -7,3 +9,20 @@ class Artist(models.Model):
...
+    title = models.CharField(max_length=200, blank=True, null=True)
+    creators = models.ManyToManyField(Artist, related_name="works")
+    year = models.IntegerField(blank=True, null=True)
+    latitude = models.DecimalField(max_digits=9, decimal_places=6, blank=True, null=True)
+    longitude = models.DecimalField(max_digits=9, decimal_places=6, blank=True, null=True)
+    category = models.CharField(max_length=20, blank=True, null=True)
+    description = models.TextField(blank=True, null=True)
+    # artistStatement = soundbyte?
+    photo = models.ImageField
+    artistWebsite = models.URLField(blank=True, null=True)
```
Comment (brownti159) [inline places/models.py]: Artist website is not needed  because the field is under the Artist class.

File: places/models.py [hunk 2]
```diff
@@ -7,3 +9,20 @@ class Artist(models.Model):
```
Comment (brownti159) [inline places/models.py:9]: A null=True needs to be added to the website field.

**Review by brownti159 — CHANGES_REQUESTED** [review]

File: places/models.py (hunk 1 above)
Comment (allisondurham) [reply places/models.py]: Great point, fixed it!

File: places/models.py (hunk 2 above)
Comment (allisondurham) [reply places/models.py:9]: Good catch!  Got it!

**Review by brownti159 — APPROVED** [review]
🦙

**Pull request merged**